src/ripple/basics/impl/contract.cpp

```diff
@@ -39,6 +41,18 @@ accessViolation() noexcept
 
 } // detail
 
+void
+LogThrow (std::string title)
+{
+    if (auto j = debugJournal().debug())
+    {
+        j << title;
+        for (auto const& frame : beast::getStackBacktrace ())
+            j << "    " << frame;
+        j << "----";
+    }
+}
```

Comment:
Oh.  Thanks.  I hadn't understood that distinction.  I thought `--debug` affected the log level.  Thanks for straightening me out.

Still.  The point of this exercise is to find out why our production servers are crashing.  So we'll want to enable this logging on our production servers (except our validators, which aren't crashing), right?  So in order for this change to be useful our production severs must pay the price of logging a stack trace for every `Throw` and `Rethrow`.

I'm not saying don't do it.  I'm just saying it's compute expensive and there is a less compute expensive and more thorough solution (since it catches all exceptions).  But that less compute expensive solution requires more coding work than this one requires.

To be clear, I'm okay with what you're doing.  I'm just presenting what I suspect is a better alternative.